src/pages/our-people.astro

```diff
@@ -0,0 +1,160 @@
+---
+import { SITE } from '~/config.mjs';
+
+import Layout from '~/layouts/PageLayout.astro';
+
+import { getCanonical, getPermalink, getRelativeLink } from '~/utils/permalinks';
+
+const meta = {
+  title: `Our People — ${SITE.name}`,
+  description: SITE.description,
+  canonical: getCanonical(getPermalink('our-people')),
+};
+
+type Person = {
+  name: string;
+  position?: string;
+  about: string;
+  icon?: string;
+};
+
+const team: Person[] = [
+  {
+    name: 'Leon Bowie',
+    position: 'Head of Operations',
+    about:
+      'Actively involved in technolgoy outreach for the past 7 years running Code Clubs, coaching robotics teams and working in industry as a DevOps Engineer. One of the founders who now runs Tūhura Tech working with our staff, volunteers and sponsors to make sure all kids can get the best support possible.',
```

Comment:
Fix typo - perhaps also you should add "Leon is" before the actively involved part for clarity/cleaner-flowing reading
```suggestion
      'Actively involved in technology outreach for the past 7 years running Code Clubs, coaching robotics teams and working in industry as a DevOps Engineer. One of the founders who now runs Tūhura Tech working with our staff, volunteers and sponsors to make sure all kids can get the best support possible.',
```